src/WindowsDebugLauncher/WindowsDebugLauncher.csproj

```diff
@@ -15,41 +13,17 @@
     <StartAction>Program</StartAction>
     <OutputPath>$(MIDefaultOutputPath)\vscode</OutputPath>
     <DropSubDir>vscode</DropSubDir>
-    <TargetFramework>net462</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
     <OutputType>Exe</OutputType>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Desktop.Debug|AnyCPU'">
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <Externalconsole>true</Externalconsole>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Desktop.Release|AnyCPU'">
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
     <Externalconsole>true</Externalconsole>
   </PropertyGroup>
-  
-  <ItemGroup Label="References">
-    <PackageReference Include="Microsoft.VisualStudioEng.MicroBuild.Core" Version="$(Microsoft_VisualStudioEng_MicroBuild_Core_Version)" GeneratePathProperty="true" />
-  </ItemGroup>
-  
+
   <ItemGroup>
-    <DropSignedFile Condition="'$(IsXPlat)' == 'true'" Include="$(OutputPath)\WindowsDebugLauncher.exe" />
+    <DropSignedFile Condition="$([MSBuild]::IsOSPlatform('windows'))" Include="$(OutputPath)\WindowsDebugLauncher.exe" />
```

Comment:
>exe [](http://example.com/codeflow?start=114&length=3)

Don't you want '.dll' here?

Reply:
Alternatively, if getting WindowsDebugLaunch.exe and OpenDebugAD7.exe to share a single self-contained .NET runtime proves tricky you could leave this project on desktop .NET.

Reply:
Does not seem to be supported yet: https://github.com/dotnet/runtime/issues/53834